### PR TITLE
[CFE-476]: feat(utils): migrate query msg from api to lcd

### PR DESCRIPTION
**Pull request by Poafs1**

### CODE REVIEW

File: src/lib/services/wasm/contract/lcd.ts [hunk 1]
```diff
@@ -37,3 +41,39 @@ export const getContractsByCodeIdLcd = (
       }
     )
     .then(({ data }) => parseWithError(zContractsResponseLcd, data));
+
+export const getContractQueryMsgs = async (
+  endpoint: string,
+  contractAddress: BechAddr32
+) => {
+  const msg = '{"": {}}';
+  const encodedMsg = Buffer.from(msg).toString("base64");
```
[the remark this answers is not in the document]
Comment (songwongtp) [reply src/lib/services/wasm/contract/lcd.ts:50]: `libEncode` *

**Comment by evilpeach** [comment]
Please check this contract osmo10c8y69yylnlwrhu32ralf08ekladhfknfqrjsy9yqc9ml8mlxpqq2sttzk
There's some query messages: https://dev.celat.one/osmosis-1/contracts/osmo10c8y69yylnlwrhu32ralf08ekladhfknfqrjsy9yqc9ml8mlxpqq2sttzk

**Review by evilpeach — APPROVED** [review]

File: src/lib/services/wasm/contract/lcd.ts [hunk 2]
```diff
@@ -37,3 +41,38 @@ export const getContractsByCodeIdLcd = (
       }
     )
     .then(({ data }) => parseWithError(zContractsResponseLcd, data));
+
+export const getContractQueryMsgs = async (
```
Comment (songwongtp) [inline src/lib/services/wasm/contract/lcd.ts:45]: getContractQueryMsgsLcd

File: src/lib/services/wasm/contract/index.ts [hunk 3]
```diff
@@ -157,7 +157,7 @@ export const useContractTableCounts = (contractAddress: BechAddr32) => {
 };
 
 export const useContractQueryMsgs = (contractAddress: BechAddr32) => {
```
Comment (songwongtp) [inline src/lib/services/wasm/contract/index.ts:159]: ...Lcd

**Review by songwongtp — APPROVED** [review]

**Pull request merged**